src/test/test-firewall-util.c

```diff
@@ -9,34 +9,50 @@
 int main(int argc, char *argv[]) {
         int r;
         test_setup_logging(LOG_DEBUG);
+        uint8_t prefixlen = 32;
 
         r = fw_add_masquerade(true, AF_INET, NULL, 0);
+        if (r == 0)
+                log_error("%s:%d Expected failure: NULL source", __FILE__, __LINE__);
```

Comment:
please drop the `__FILE__` + `__LINE__` stuff. You get the same automatically for all log messages by setting the `SYSTEMD_LOG_LOCATION` env var to `1`. Please, let's not add redundant ways to get the sam ehere...